CHANGELOG.md

```diff
@@ -8,6 +8,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 The intended audience of this file is for py42 consumers -- as such, changes that don't affect
 how a consumer would use the library (e.g. adding unit tests, updating documentation, etc) are not captured here.
 
+## Unreleased
+
+### Added
+
+- `code42 legal-hold events` command:
```

Comment:
Should the command be called `search-events` instead?

* It'd be a verb, like the rest of our commands
* It'd mirror the `file-events search` command

Reply:
Changed to `search-events` > let me know if this is not what you had in mind